ci/poll/main.go

```diff
@@ -0,0 +1,41 @@
+/*
+|    Protect your secrets, protect your sensitive data.
+:    Explore VMware Secrets Manager docs at https://vsecm.com/
+</
+<>/  keep your secrets… secret
+>/
+<>/' Copyright 2023–present VMware, Inc.
+>/'  SPDX-License-Identifier: BSD-2-Clause
+*/
+
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	if err := createLockFile(); err != nil {
+		fmt.Println("Another instance is already running. Exiting.")
+		return
+	}
+
+	defer removeLockFile()
+
+	move, currentCommitHash := proceed()
+	if !move {
+		return
```

Comment:
(Minor) We can log error here before returning, same with runPipeline()

Reply:
Noted… will put in a follow-up PR.